docs/reference/rdx.md

```
@@ -1,248 +1,493 @@
+The `--output` path MUST be written using standard file I/O. `rdx` MUST NOT
+create parent directories for `--output` paths; missing parent directories
+cause an I/O error and exit `1`. Existing files MUST be overwritten.
+
+The `--output` flag MUST have no effect in `--check` mode.
+
+<a id="9-failure-modes"></a>
+
+## 9. Failure Modes
+
+`rdx` MUST exit with status `1` and write a single-line error to stderr,
+prefixed by `error: `, in each of the following cases.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)